addons/scratchblocks/userscript.js

```diff
@@ -44,16 +44,25 @@ async function getLocales(addon) {
 export default async function ({ addon, global }) {
   window.scratchAddons._scratchblocks3Enabled = true;
   const blocks = document.querySelectorAll("pre.blocks");
-  blocks.forEach((block) => {
-    block.textContent = block.getAttribute("data-original");
-  });
 
   // Translations can't load first
   await addon.tab.loadScript(addon.self.lib + "/thirdparty/cs/scratchblocks-v3.5.2-min.js");
   await addon.tab.loadScript(addon.self.lib + "/thirdparty/cs/translations-all-v3.5.2.js");
 
   const languages = await getLocales(addon);
 
+  if (blocks.length > 0) {
```

Comment:
I got `blocks.length === 0` at this line (remember: this addon can load before DOM!)

Reply:
Now it gets the blocks after the DOM is loaded, try now?